Changed JWT Expiration to ID Token Expiration

tamigoodall opened the pull request:
Changed term from JWT Expiration to ID Token Expiration
Replaced some screenshots
Removed unnecessary dashboard screenshots

<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors
- Please include a short description
- If your PR is a work in progress, title it [DO NOT MERGE]
--->

tamigoodall committed:
Changed JWT Expiration to ID Token Expiration

Changed in 6 files
removed unnecessary dashboard screenshot

madebyankur commented:
Looks good to me. I found this other instance that should be updated (including the URI) -
https://auth0.comdocs/best-practices/application-settings#set-jwt-token-expiration

tamigoodall committed:
Update application-settings.md

Cleaned up best practices list for app settings
Changed JWT Token expiration to ID Token expiration

madebyankur reviewed (dismissed):
LGTM

tamigoodall committed:
Replaced screenshots

Replaced screenshots

Commit:
Minor copyedits
